validators/RequestSchema.js

```diff
@@ -145,6 +145,35 @@ const DeleteMediaFileRequestSchema = Joi.object().keys({
   sha: Joi.string().required(),
 })
 
+const UpdateNavigationRequestSchema = Joi.object().keys({
+  content: Joi.object()
+    .keys({
+      logo: Joi.string().allow(""),
+      links: Joi.array()
+        .items(
+          Joi.object()
+            .keys({
+              title: Joi.string().required(),
+              url: Joi.string(),
+              collection: Joi.string(),
+              resource_room: Joi.boolean().valid(true),
+              sublinks: Joi.array().items(
+                Joi.object().keys({
+                  title: Joi.string().required(),
+                  url: Joi.string().required(),
+                })
+              ),
+            })
+            .oxor("url", "collection", "resource_room")
+            .oxor("sublinks", "collection")
+            .oxor("sublinks", "resource_room")
```

Comment:
i think we might wanna check that joi returns a good error message here or if it doesn't, set the message ourselves using `.message` so that when the FE displays the message to users, they aren't confused

Reply:
As discussed offline, since our schema is used more as a test of correctness, we don't expect this error to be propagated to the frontend in normal scenarios, so we're opting to leave it as is for now